.github/workflows/deploy_qa.yml

```diff
@@ -0,0 +1,45 @@
+name: deploy qa
+
+on:
+  push:
+    branches:
+      - qa
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  AWS_DEFAULT_REGION: us-east-1
+  LOCAL_TAG_NAME: dc-frontend:${{ github.ref_name }}-${{ github.run_number }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::685731035297:role/GithubActionsDeployerRole
+          aws-region: us-east-1
+
+      - name: Log in to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build and push Docker image to Amazon ECR
+        env:
+          APP_ENV: qa
+          NEW_RELIC_LICENSE_KEY: ${{ secrets.NEW_RELIC_LICENSE_KEY }}
+          ECR_URL: 685731035297.dkr.ecr.us-east-1.amazonaws.com/dc-frontend:qa-latest
```

Comment:
I usually see the final ECR url as 
```
ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
ECR_REPOSITORY: dc-frontend
```

and then 

```
docker push $ECR_REGISTRY/$ECR_REPOSITORY:qa-latest
```

I _think_ this should also work. You can leave this it is, but if there are any issues deploying, this is a place to start.